### PR TITLE
Fix the bug of status indictor using string comparison instead of number

### PR DESCRIPTION
**Pull request by xizi-xu**

### CODE REVIEW

File: ArgusWeb/app/js/services/charts/chartRendering.js [hunk 1]
```diff
@@ -48,15 +48,19 @@ angular.module('argus.services.charts.rendering', [])
...
-								$('#' + attributes.name + '-status').removeClass('red orange green').addClass('red');
-						} else if (lastStatusVal > attributes.lo && lastStatusVal < attributes.hi) {
-								$('#' + attributes.name + '-status').removeClass('red orange green').addClass('orange');
-						} else if (lastStatusVal > attributes.hi) {
-								$('#' + attributes.name + '-status').removeClass('red orange green').addClass('green');
-						}
+                    var lastStatusValNum = Number(lastStatusVal);
+                    var lowValNum = Number(attributes.lo);
+                    var highValNum = Number(attributes.hi);
+
```
Comment (bsura) [inline ArgusWeb/app/js/services/charts/chartRendering.js]: Although this wouldn't generate any errors, I would be nice to log to console if attributes.lo or attributes.hi is undefined. That way one can easily check the console and figure out that hi and lo are undefined. Makes sense?

Comment (xizi-xu) [inline ArgusWeb/app/js/services/charts/chartRendering.js]: Do we want to have a growl message as well when lo or high is not defined?

Comment (xizi-xu) [inline ArgusWeb/app/js/services/charts/chartRendering.js]: @bsura

Comment (bsura) [inline ArgusWeb/app/js/services/charts/chartRendering.js]: Umm. Growl would be too much I suppose. If it were me, I would probably just console.log() it for the purpose of debugging. I would leave that up to you.

**Pull request merged**